projects/plugins/jetpack/extensions/blocks/yoast-promo/index.js

```diff
@@ -0,0 +1,106 @@
+import { PanelBody, PanelRow, ExternalLink, Button } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { PluginPrePublishPanel } from '@wordpress/edit-post';
+import { __ } from '@wordpress/i18n';
+import Gridicon from 'gridicons';
+import { useState } from 'react';
+import JetpackPluginSidebar from '../../shared/jetpack-plugin-sidebar';
+import { JetpackYoastLogos } from './JetpackYoastLogos';
+
+const PLUGIN_SLUG_YOAST_FREE = 'wordpress-seo/wp-seo';
+const PLUGIN_SLUG_YOAST_PREMIUM = 'wordpress-seo-premium/wp-seo-premium';
+
+export const YoastPromo = () => {
+	const [ isDismissed, setIsDismissed ] = useState( false );
+	const { isYoastFreeActive, isYoastPremiumActive } = useSelect( select => {
+		const { getPlugin } = select( 'core' );
+
+		const isPluginActive = slug => {
+			const plugin = getPlugin( slug );
+			return plugin && 'active' === plugin.status;
+		};
+
+		return {
+			isYoastFreeActive: isPluginActive( PLUGIN_SLUG_YOAST_FREE ),
+			isYoastPremiumActive: isPluginActive( PLUGIN_SLUG_YOAST_PREMIUM ),
+		};
+	} );
+
+	const handleDismiss = () => setIsDismissed( true );
```

Comment:
Should we set this as a more permanent dismissal, so folks do not keep seeing it with every new post they create once they've dismissed it once?

Reply:
Good idea, I'll use local storage implementation for this purpose now.